extensions/interactions/SetInput/directives/set-input-validation.service.ts

```diff
@@ -34,13 +37,124 @@ export class SetInputValidationService {
     private baseInteractionValidationServiceInstance:
       baseInteractionValidationService) {}
 
+  private isSameSet(inputA: string[], inputB: string[]): boolean {
+    let setA = new Set(inputA);
+    let setB = new Set(inputB);
+    if (setA.size !== setB.size) {
+      return false;
+    }
+    for (let element of setA.values()) {
+      if (!setB.has(element)) {
+        return false;
+      }
+    }
+    return true;
```

Comment:
You can just call `isSubset` here instead, since we already know that the sets are the same length.

Reply:
Done